### ISSUE TITLE
recent commits cause merged models to lose model layers, rendering them untrainable in kohya_ss

### ISSUE THREAD

**Issue by SQCU** [hako-mikan/sd-webui-supermerger#255]
https://github.com/hako-mikan/sd-webui-supermerger/commit/64a40445b25e49f3cac3f24b3e7b338e3caa258f
isolated problem to a commit from the same day as this.
reverting to d5decbb fixes the problem.
attaching examples of the messed up merged models in the sd-webui-model-toolkit extension and in training scripts.

because the issue has already been resolved (by reverting to a previous version) I do not think any more information is needed to address the problem for other users.
![firefox_2023-09-27_16-29-11](https://github.com/hako-mikan/sd-webui-supermerger/assets/70545818/a8f0ba6f-6e4d-403f-85ec-8c3cc4c64b92)
![cmd_2023-09-27_23-50-15](https://github.com/hako-mikan/sd-webui-supermerger/assets/70545818/09a534d7-a0d9-41c4-99d1-2c76a371adf6)

**Comment by hako-mikan** [comment]
So when merging the two models, a specific key gets lost. Does this phenomenon occur with all models?

**Comment by SQCU** [comment]
Yes, this error I experienced affected all of the different models I
attempted merging. Only keys from the clip models were ever lost, and every
model I merged would produce a valid U-net and Vae model with all of the
necessary keys.

I am away from my inference computer at the moment and cannot retrieve all
of my notes, but I have several working supermerged U-nets from these
corrupted merges. These particular u-nets work as expected if they are
substituted as the u-net portion of another working model.
